pkgs/development/perl-modules/findimagedupes/default.nix

```diff
@@ -0,0 +1,70 @@
+{ lib, stdenv, fetchurl, makeWrapper, perl, perlPackages, installShellFiles }:
+
+stdenv.mkDerivation rec {
+  pname = "findimagedupes";
+  version = "2.19.1";
+
+  # fetching this from GitHub does not contain the correct version number
+  src = fetchurl {
+    url = "http://www.jhnc.org/findimagedupes/findimagedupes-${version}.tar.gz";
+    sha256 = "sha256-5NBPoXNZays5wzpQYar4uZZb0P/zB7fdecE+SjkJjcI=";
+  };
+
+  # Work around the "unpacker appears to have produced no directories"
+  setSourceRoot = "sourceRoot=$(pwd)";
+
+  nativeBuildInputs = [ makeWrapper installShellFiles ];
+
+  buildInputs = [ perl ] ++ (with perlPackages; [
+    DBFile
+    FileMimeInfo
+    FileBaseDir
+    #GraphicsMagick
+    ImageMagick
+    Inline
+    InlineC
+    ParseRecDescent
+  ]);
+
+  # use /tmp as a storage
+  # replace GraphicsMagick with ImageMagick, because perl bindings are not yet available
+  postPatch = ''
+    substituteInPlace findimagedupes \
+      --replace "DIRECTORY => '/usr/local/lib/findimagedupes';" "DIRECTORY => '/tmp';" \
+      --replace "Graphics::Magick" "Image::Magick"
+  '';
+
+  buildPhase = "
+    runHook preBuild
+    ${perl}/bin/pod2man findimagedupes > findimagedupes.1
```

Comment:
We cannot put perl into nativeBuildInputs?

Reply:
same result as above:
```
/home/user/.nix-profile/bin/findimagedupes: /nix/store/5w6rgxdanp5iszg3rfqgfx9131drxixj-findimagedupes-2.19.1/bin/.findimagedupes-wrapped: /usr/bin/perl: bad interpreter: No such file or directory
/home/user/.nix-profile/bin/findimagedupes: line 3: /nix/store/5w6rgxdanp5iszg3rfqgfx9131drxixj-findimagedupes-2.19.1/bin/.findimagedupes-wrapped: Success
```

Reply:
since the script itself is a perl-script. it will always be needed.